index.rst

```diff
@@ -350,12 +410,12 @@ For other components, see the exception section below.
 
 #.  Edit/Add/Replace the configuration file(s) in the CSC's configuration directory.
 
-        - If editing the ``_init.yaml`` or a ``_<site>.yaml`` file, the filename must remain unchanged.
-        - If editing or adding an configuration override file, ideally the name of the file should reflect the purpose of change; dates can also be used as well.
-          Old configuration files can be kept in the repo if they still represent valid configurations. Otherwise, they should be removed.
-          Note, though, that they will still remain available on previous commits in the git repo, enabling historical comparison.
+    - If editing the ``_init.yaml`` or a ``_<site>.yaml`` file, the filename must remain unchanged.
+    - If editing or adding a configuration override file, ideally the name of the file should reflect the purpose of change; dates can also be used as well.
```

Comment:
Please remove "ideally", to strengthen the recommendation. 

Also consider "You may find it useful to include a date as part of the name."